cmd/podman/generate/systemd.go

```diff
@@ -46,6 +46,7 @@ func init() {
 	flags := systemdCmd.Flags()
 	flags.BoolVarP(&systemdOptions.Name, "name", "n", false, "Use container/pod names instead of IDs")
 	flags.BoolVarP(&files, "files", "f", false, "Generate .service files instead of printing to stdout")
+	flags.BoolVarP(&systemdOptions.TemplateUnitFile, "template", "", false, "Make it a template file and use %i and %I specifiers")
```

Comment:
Should we add `(only works on containers)`?

Reply:
DONE :+1: